### PR TITLE
Rework concepts with Post Bahmni module

### PR DESCRIPTION
**Pull request by reagan-meant**

### CODE REVIEW

File: configuration/ordertypes/order_types.csv [hunk 1]
```diff
@@ -1,8 +1,2 @@
 Uuid,Void/Retire,Name,Description,Java class name,Parent,Concept Classes,_order:100
-ef83e09d-8885-4f12-be6b-06a80aa62972,,Product,An order for a product,org.openmrs.Order,,Product,
-,True,Radiology Order,,,,,
-5338a5b1-2cbc-4081-9a9b-9e479e2acaad,,Radiology Order,An order for radiology,org.openmrs.Order,,Radiology,
-,True,Drug Order,,,,,
-16bb6bcf-2e81-4ba5-8583-ff4913201a72,,Drug Order,An order for a medication to be given to the patient,org.openmrs.DrugOrder,,Drug,
-,True,Lab Order,,,,,
-f8ae333e-9d1a-423e-a6a8-c3a0687ebcf2,,Lab Order,An order for laboratory tests,org.openmrs.Order,,LabTest;LabSet,
```
Comment (rbuisson) [inline configuration/ordertypes/order_types.csv]: Are you sure we should remove these?

Comment (reagan-meant) [reply configuration/ordertypes/order_types.csv]: These are built added by Bahmni except product which I have left....So I just edited them

Comment (rbuisson) [reply configuration/ordertypes/order_types.csv]: Additions mades by Bahmni are not to be trusted, and as we know, we might not use Bahmni tmr. The config should ensure they are created.
If they are already there, then no worries, not much will happen (assuming we know their UUID)

Comment (reagan-meant) [reply configuration/ordertypes/order_types.csv]: uuids are dynamic 
https://github.com/Bahmni/bahmni-core/blob/8c6caf014d2c683a1a54dfed23deb5df75753cfa/bahmnicore-omod/src/main/resources/V1_55__addLabAndRadiologyOrderTypes.sql#L1

Comment (rbuisson) [reply configuration/ordertypes/order_types.csv]: So we should probably add this to post bahmni first?

**Pull request merged**